app/models/king.rb

```diff
@@ -10,4 +10,11 @@ def valid_move?(destination_x, destination_y)
     return valid
   end
 
+  def unicode_symbol
+    if "white"
```

Comment:
Added that on one piece - queen - and both queens turned black. The Unicode is correct. We have 'white' and 'black' to use - they are defined in the Piece.rb. Maybe I should do a when, case statement?